lib/clients/incoming-webhook/client.js

```diff
@@ -69,7 +74,11 @@ IncomingWebhook.prototype._formatData = function _formatData(message) {
   } else if (isObject(message)) {
     if (message.text) data.text = message.text;
     if (message.username) data.username = message.username;
-    if (message.iconEmoji) data.icon_emoji = message.iconEmoji;
+    if (message.iconUrl) {
```

Comment:
According to the api documentation the icon_emoji argument overrides the icon_url argument when present. I think that this if statement should be changed to mirror the api more closely. [https://api.slack.com/methods/chat.postMessage](https://api.slack.com/methods/chat.postMessage)